reactor-netty-http/src/test/java/reactor/netty/http/client/HttpClientTest.java

```diff
@@ -614,6 +618,97 @@ void sslExchangeRelativeGet() throws SSLException {
 		assertThat(responseString).isEqualTo("hello /foo");
 	}
 
+	@Test
```

Comment:
IMO we can simplify these tests with using `@ParameterizedTest`

Something like the snippet below. Wdyt?

```
@ParameterizedTest
@ValueSource(booleans = {true, false})
void maxConnectionPools(boolean withMaxConnectionPools) throws SSLException {

...
		ConnectionProvider connectionProvider = withMaxConnectionPools ?
				ConnectionProvider.builder("max-connection-pool").maxConnectionPools(1).build() :
				ConnectionProvider.builder("max-connection-pool").build();
...

}
```